### PR TITLE
Add run_dbcan

### PR DESCRIPTION
**Pull request by lecorguille**
Add run_dbcan
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by lecorguille** [comment]
@BiocondaBot please add label

File: recipes/run-dbcan/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: GPL3
+  license_file: LICENSE 
+  summary: "Standalone version of dbCAN annotation tool for automated CAZyme annotation"
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (bgruening) [inline recipes/run-dbcan/meta.yaml:48]: can you add your name here or remove it? Is there a bio.tool ID?

File: recipes/run-dbcan/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - diamond
+    - hmmer
+
+test:
+  commands:
+    - run_dbcan.py --help
+
+about:
+  home: "https://github.com/linnabrown/run_dbcan"
+  license: "GNU General Public v3 (GPLv3)"
```
Comment (bgruening) [inline recipes/run-dbcan/meta.yaml:39]: can you please use spdx identifiers? https://spdx.org/licenses/GPL-3.0-only.html

File: recipes/run-dbcan/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  license_family: GPL3
+  license_file: LICENSE 
+  summary: "Standalone version of dbCAN annotation tool for automated CAZyme annotation"
+  doc_url:
```
Comment (bgruening) [inline recipes/run-dbcan/meta.yaml:43]: add or remove :)

File: recipes/run-dbcan/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "run-dbcan" %}
+{% set version = "2.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/ae/87/431b9b651544ead5b0889c39e6fddb9e1dd98e773c2ed5c229e18e063ee6/run_dbcan-2.0.11.tar.gz"
```
Comment (bgruening) [inline recipes/run-dbcan/meta.yaml:9]: If you use this URL https://pypi.io/packages/source/r/run_dbcan/run_dbcan-2.0.11.tar.gz and with the version variable {{version}} than the bot can update it easily

Comment (lecorguille) [reply recipes/run-dbcan/meta.yaml]: The link was down yesterday. It works now!

File: recipes/run-dbcan/meta.yaml (hunk 3 above)
Comment (lecorguille) [reply recipes/run-dbcan/meta.yaml]: Done

File: recipes/run-dbcan/meta.yaml (hunk 1 above)
Comment (lecorguille) [reply recipes/run-dbcan/meta.yaml]: No bio.tool ID

**Review by bgruening — APPROVED** [review]
Thanks.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**